### ISSUE TITLE
Licence for generated images

### ISSUE THREAD

**Issue by inactinique** [cutterkom/generativeart#8]
Dear all,

We are using the generativeart code to design the cover of an academic open access journal.
We found the licence for the code. But what about the images themselves? Can they be published with the same licence or, rather, a corresponding CC licence?

Best,
Frédéric Clavert

**Comment by cutterkom** [comment]
Hello Frédéric, 

sorry for the delayed answer. A very interesting question, indeed. Actually, I never really thought about it. Since I published the package under GPL3, I assume the resulting images are "free", too. 

I would have suggested citing the package, but I saw on your website (https://www.c2dh.uni.lu/news/digital-history-journal-transmedia-storytelling-hermeneutics-data) you did that anyway. Thanks!

Best
Katharina
